swag/client/ui/IHS_toolkit.py

```diff
@@ -0,0 +1,55 @@
+from typing import TYPE_CHECKING
```

Comment:
Lowercase for this file name would be better

Reply:
Resolved in 62c725d ("Renaming IHS_toolkit by ihs_toolkit", 2022-02-21)